cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java

```diff
@@ -407,6 +409,12 @@ private void performUpgrade() throws Exception {
     QueueConfigUpgrader queueConfigUpgrader = injector.getInstance(QueueConfigUpgrader.class);
     queueConfigUpgrader.upgrade();
 
+    // Upgrade all stream and queue tables.
+    StreamAdmin streamAdmin = injector.getInstance(StreamAdmin.class);
```

Comment:
should we do this in ``QueueConfigUpgrader.upgrade`` and ``StreamStateStore.upgrade``?